decision-records/0001-schema-improvements.md

```diff
@@ -0,0 +1,114 @@
+# Proposal for Schema Improvements
+
+- Date: 2024-10-22
+
+## Introduction
+
+This RFC proposes improvements to the current data schema to improve usability. Since changing the data structure entails publishing a new major version—which can be costly for downstream dependencies—it is crucial to use this opportunity to reconsider the entire vocabulary and structure.
+
+## Context and Problem Statement
+
+### Confusion around the `writer` field
+
+The `writer` field is filled in with "service provider" in 32 out of 41 terms types, which does not seem to add significant value. Other values, such as `transportation operator (airline, railway, bus…)` (once), `paid-for goods or service provider` (once), `online marketplace` (twice), `search engine or intermediation service provider` (twice), and `intermediation service provider` (three times), can provide information about the industry in which the terms operate, which is not the original purpose of the `writer` field.
+
+### Ambiguity in the `audience` field
+
+There is ambiguity between the audience of the document and the audience of the service. Currently, the `audience` field could be interpreted in these two ways.
+
+### Complex vocabulary
+
+Several fields are named in ways that could be clearer, such as the `object` field of the `commitment`.
+
+## Proposed solutions
+
+### Solution A
+
+#### Schema
+
+```json
+"<terms type name>": {
+  "defines": "...",
+  "used by": "...", // Default value: "service provider"
+  "addressed to": "...", // Default value: "end user"
+  "also known as": [ // Optional
+    "..."
+  ],
+  "referenced by": { // Optional
+    "name": "url"
+  }
+}
+```
+
+##### Examples
+
+```json
+"Editorial Policy": {
+  "defines": "the writing and publishing standards and principles",
+  "also known as": [
+    "Editorial Guidelines"
+  ],
+  "referenced by": {
+    "Open Terms Archive discussion": "https://github.com/OpenTermsArchive/terms-types/discussions/35"
+  }
+}
+```
+
+```json
+"Conditions of Carriage": {
+  "defines": "benefits and limitations associated with the transportation being provided",
+  "used by": "transportation operator (airline, railway, bus…)",
+  "addressed to": "passenger",
+  "also known as": [
+    "Transport Conditions"
+  ]
+}
+```
+
+### Solution B 
+
+#### Schema
+
+
+```json
+"<terms type name>": {
+  "subject": "...",
+  "owner": "...", // Default value: "service provider"
+  "recipient": "...", // Default value: "end user"
```

Comment:
“recipient” implies an intention to _send_ the information to someone. I feel this is more about having a _target_, a _reader_ or an _audience_…

Reply:
I vote for `audience`.

Reply:
Or maybe `intended users`?

Reply:
I understand. From my point of view, the `audience` is too ambiguous, `target` has a less appropriate semantic field. 

`reader` seems fine to me.

Reply:
I'm not in favor of the term `reader`. While I can read this document, being a 'reader' doesn't necessarily mean I am concerned by its content.

Reply:
@clementbiron what about `intended users`?

Reply:
Looking through the thesaurus, what about “party”? It is unfortunately ambiguous with a fun event, but it is relevant in the meaning “parties to an agreement”.
Another possibility could be “contractor”, but it is also ambiguous with some entity that procures some service.
I also discovered the word “obligor”: “a person who is bound to another by contract or other legal procedure”.

Reply:
Among all propositions, for now I still vote for `audience`.
`party`, `contractor` or `obligor` are not very clear for me. 
Moreover, not all our terms type involve the same level of commitment. An agreement does not imply the same level of commitment as a guideline. So `contractor` and words with this kind of meaning seems to restrictive for me.

Reply:
Otherwise, what about `stakeholders`?

Reply:
`intended users` is very clear but I struggle a bit with the fact that it's in two words